client/rest/src/main/java/org/opensearch/client/RestClient.java

```diff
@@ -431,6 +437,48 @@ private Publisher<Message<HttpResponse, Publisher<ByteBuffer>>> streamRequest(
         });
     }
 
+    /**
+     * Frame the {@link ByteBuffer} into individual chunks that are separated by '\r\n' sequence.
+     * @param b {@link ByteBuffer} to split
+     * @return individual chunks
+     */
+    private static Collection<ByteBuffer> frame(ByteBuffer b) {
```

Comment:
This looks like a good candidate for a unit test. What do you think?

Reply:
My apologies, have seen the comment after merge only. Sadly our `codecov` captures only limited subset of tests, the `ReactorNetty4StreamingIT` covers this method in full:

![image](https://github.com/user-attachments/assets/36cb99b1-5911-4bf5-8f4e-33c708a42383)